recipes/recipes/emscripten_emscripten-32/recipe.yaml

```diff
@@ -33,6 +33,8 @@ outputs:
           - "{{ pin_compatible('emscripten-abi', min_pin='x.x.x', max_pin='x.x.x') }}"
 
     requirements:
+      build:
```

Comment:
Shouldn't the build number be incremented as well?

Reply:
Seeing some recent issues on the `voici` CI which tend to hint at this change:

```
Transaction starting
error    libmamba File not valid: file size doesn't match expectation "/tmp/xeus-python-kernel/pkgs/emscripten-abi-3.1.27-hb0f4dca_5.tar.bz2"
    Expected: 4324
    Actual: 4298
    
emscripten-abi-3.1.27-hb0f4dca_5.tar.bz2 tarball has incorrect size
warning  libmamba '/tmp/xeus-python-kernel/pkgs/emscripten-abi-3.1.27-hb0f4dca_5.tar.bz2' validation failed
critical libmamba Found incorrect download: emscripten-abi. Aborting
```

Reply:
yes, sry for not catching that

here is a fixup https://github.com/emscripten-forge/recipes/pull/459